notebooks/01_prepare_data/wikidata.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>adding rows with concat is very inneficient, it is faster to use loc. See an example here: <a href="https://github.com/miguelgfierro/pybase/blob/1ce06734e236fc5367c6b946b16e4820b0cd84d0/pandas_base/conversion.py#L192" target="_blank">https://github.com/miguelgfierro/pybase/blob/1ce06734e236fc5367c6b946b16e4820b0cd84d0/pandas_base/conversion.py#L192</a></p>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/778/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>